src/actions/channels.js

```diff
@@ -1122,11 +1139,23 @@ export function updateChannelPurpose(channelId: string, purpose: string): Action
     };
 }
 
+function isManuallyUnread(channelId: ?string, getState: () => GlobalState): boolean {
```

Comment:
this does not look like an action, maybe consider moving it to a utils file.

Reply:
sounds ok! moved to helpers.